cmd/prometheus/main.go

```diff
@@ -1201,6 +1206,7 @@ type tsdbOptions struct {
 func (opts tsdbOptions) ToTSDBOptions() tsdb.Options {
 	return tsdb.Options{
 		WALSegmentSize:         int(opts.WALSegmentSize),
+		MaxChunkSize:           int64(opts.MaxChunkSize),
```

Comment:
```suggestion
		MaxChunkSize:           int(opts.MaxChunkSize),
```

Reply:
The underlying field this ends up controlling in chunks.go is int64, so feels like we should probably match type through the rest of the stack.

Reply:
I was just confused by the fact we might not need so big chunks at the current state of IT systems and the fact that another size is expressed in int on the line above :)